.travis.yml

```diff
@@ -23,6 +23,7 @@ branches:
 
 before_install:
   - git config --global url."https://$GHE_USER:$GHE_TOKEN@github.com/".insteadOf "https://github.com/"
+  - git config --global url."ssh://git@github.ibm.com".insteadOf https://github.ibm.com
```

Comment:
Remove the github.ibm.com config

Reply:
If we remove this then oss task wil fail we need this